exclude_conformance_test.txt

```diff
@@ -0,0 +1,39 @@
+[sig-storage] ConfigMap  should be consumable from pods in volume as non-root [NodeConformance] [Conformance]
+[sig-storage] ConfigMap  should be consumable from pods in volume with defaultMode set [NodeConformance] [Conformance]
+[sig-storage] ConfigMap  should be consumable from pods in volume with mappings and Item mode set [NodeConformance] [Conformance]
+[sig-storage] ConfigMap  should be consumable from pods in volume with mappings as non-root [NodeConformance] [Conformance]
+
+[sig-storage] Secrets  should be consumable from pods in volume as non-root with defaultMode and fsGroup set [NodeConformance] [Conformance]
+[sig-storage] Secrets  should be consumable from pods in volume with defaultMode set [NodeConformance] [Conformance]
+[sig-storage] Secrets  should be consumable from pods in volume with mappings and Item Mode set [NodeConformance] [Conformance]
+
+[sig-storage] Projected  should be consumable from pods in volume as non-root [NodeConformance] [Conformance]
+[sig-storage] Projected  should be consumable from pods in volume as non-root with defaultMode and fsGroup set [NodeConformance] [Conformance]
+[sig-storage] Projected  should be consumable from pods in volume with defaultMode set [NodeConformance] [Conformance]
+[sig-storage] Projected  should be consumable from pods in volume with defaultMode set [NodeConformance] [Conformance]
+[sig-storage] Projected  should be consumable from pods in volume with mappings and Item mode set [NodeConformance] [Conformance]
+[sig-storage] Projected  should be consumable from pods in volume with mappings and Item Mode set [NodeConformance] [Conformance]
+[sig-storage] Projected  should be consumable from pods in volume with mappings as non-root [NodeConformance] [Conformance]
+[sig-storage] Projected  should set DefaultMode on files [NodeConformance] [Conformance]
+[sig-storage] Projected  should set mode on item file [NodeConformance] [Conformance]
+
+[sig-storage] Downward API volume  should set DefaultMode on files [NodeConformance] [Conformance]
+[sig-storage] Downward API volume  should set mode on item file [NodeConformance] [Conformance]
+
+"[sig-storage] EmptyDir volumes  should support (non-root,0644,default) [NodeConformance] [Conformance]"

```

Comment:
These quotes don't make it very easy for copy/paste. Nor parsing.

Reply:
Removed them. Should I remove the spaces as well?

Reply:
Neah, spaces are fine